modules/network-observability-configuring-custom-metrics.adoc

```diff
@@ -14,11 +14,8 @@ You can configure the `FlowMetric` API to create custom metrics by using flowlog
 . In the *Provided APIs* heading for the *NetObserv Operator*, select *FlowMetric*.
 . In the *Project:*  dropdown list, select the project of the Network Observability Operator instance.
 . Click *Create FlowMetric*.
-. Configure the `FlowMetric` resource, similar to the following sample configurations:
+. Configure the `FlowMetric` resource to generate a metric that tracks ingress bytes received from cluster external sources, similar to the following sample configuration:
```

Comment:
Like we discussed [here](https://github.com/openshift/openshift-docs/pull/98275#discussion_r2319536517), it's the same thing here: "a metric that tracks ingress bytes received from cluster external sources" would just be an use case example, not something to be considered in generic steps

Reply:
@gwynnemonahan how about doing something like that: 
<img width="873" height="632" alt="image" src="https://github.com/user-attachments/assets/22900cea-2d82-44fe-bf9f-2ebd07a2f9e0" />
 ?